city_scrapers/spiders/cook_county_board_ethics.py

```diff
@@ -0,0 +1,136 @@
+from datetime import datetime, timedelta
+
+import scrapy
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class CookCountyBoardEthicsSpider(CityScrapersSpider):
+    name = 'cook_county_board_ethics'
+    agency = 'Cook County Government Board of Ethics'
+    timezone = 'America/Chicago'
+    allowed_domains = ['www.cookcountyil.gov']
+
+    def start_requests(self):
+        today = datetime.now()
+        next_month = today.replace(day=28) + timedelta(days=5)
+        for dt in [today, next_month]:
```

Comment:
This should do a time range of 2 months in the past through 2 months in the future. Here's an example:

https://github.com/City-Bureau/city-scrapers/blob/3ae5c3e2304c9fb47576ba8fa71d05993e2055ae/city_scrapers/spiders/cook_housing.py#L15-L23